### PR TITLE
SO-432 - Supplier account notifications - data

### PR DESCRIPTION
**Pull request by adohall**

### CODE REVIEW

File: app/templates/seller-account.hbs [hunk 1]
```diff
@@ -36,6 +36,21 @@
         <p>Submitting changes to this information will require the review of the buy.nsw operations team and will take 2-3 working days.</p>
         <p><b>Not what you were looking for? Edit your {{ link-to "supplier profile" "profile-builder" }}.</b></p>
       </div>
+      {{#if this.model.alertingDocuments.professional_indemnity_certificate}}
```
Comment (armansa) [inline app/templates/seller-account.hbs]: Financial statement is not displayed here.
If documents are already expired it should show different copy.
If more than one document is expired, please show one box for all.
Don't repeat code for four documents, it's better to write a loop over them.

Comment (adohall) [reply app/templates/seller-account.hbs]: I've dealt with the different copy for cases in which the document has already expired. 
Now I'm rewriting as a loop inside a single box. For that I'll need to extract the name of each document from the key by removing underscores and capitalising.

File: app/templates/seller-account.hbs [hunk 2]
```diff
@@ -36,6 +36,19 @@
         <p>Submitting changes to this information will require the review of the buy.nsw operations team and will take 2-3 working days.</p>
         <p><b>Not what you were looking for? Edit your {{ link-to "supplier profile" "profile-builder" }}.</b></p>
       </div>
+      {{#if this.model.alertingDocuments}}
```
Comment (armansa) [inline app/templates/seller-account.hbs]: This condition always returns true! Please use (not (is-empty condition)) instead.
Please test the code and make sure it shows proper message in different cases.

Comment (adohall) [reply app/templates/seller-account.hbs]: Done. (not (is-empty myArray)) condition now used.

**Pull request merged**